src/components/views/dialogs/DevtoolsDialog.js

```diff
@@ -551,11 +551,55 @@ class AccountDataExplorer extends DevtoolsComponent {
     }
 }
 
+class ServersInRoomList extends DevtoolsComponent {
+    static getLabel() { return _t('View Servers in Room'); }
+
+    static propTypes = {
+        onBack: PropTypes.func.isRequired,
+    };
+
+    constructor(props, context) {
+        super(props, context);
+
+        this.onQuery = this.onQuery.bind(this);
```

Comment:
we've since changed how we do this kind of binding. Instead, it is preferred to be represented as:

```
onQuery = (query) => {
  // do stuff
};
```